Documentation/internals/security-identities.rst

```diff
@@ -0,0 +1,61 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _security_identities:
+
+*******************
+Security Identities
+*******************
+
+Security identities are generated from labels. They are stored as ``uint32``,
+which means the maximum limit for a security identity is ``2^32 - 1``. The
+minimum security identity is ``1``, due to the reserved identity range being
+from ``1`` to ``255``. See the `internal code documentation
```

Comment:
Not clear why the `reserved identity range being from 1 to 255` _causes_ the minimum value to be 1.

Reply:
Bad wording. The actual identity range starts at 1 because the first reserved ID is 1. 0 is a special case as mentioned in the note.